intera_examples/scripts/set_interaction_options.py

```diff
@@ -24,18 +24,99 @@
 import intera_interface
 from intera_interface import CHECK_VERSION
 
+class SetInteractionOptions(object):
+    """
+    Send Interaction Options to robot
+    """
+
+    def __init__(self, args):
+        self.pub = rospy.Publisher('/robot/limb/right/interaction_control_command',
+                                  InteractionControlCommand, queue_size = 1)
+        rospy.sleep(0.5)
+
+        self.repeat = False
+        if args.rate > 0:
+            self.rate = rospy.Rate(args.rate)
+            self.repeat = True
+        elif args.rate == 0:
+            rospy.logwarn('Interaction control options will be set only once!')
+        elif args.rate < 0:
+            rospy.logerr('Invalid publish rate!')
+
+        # set the interaction control options in the current configuration
+        interaction_options = InteractionOptions()
+
+        interaction_options.set_interaction_control_active(int2bool(args.interaction_active))
+        interaction_options.set_K_impedance(args.K_impedance)
+        interaction_options.set_max_impedance(int2bool(args.max_impedance))
+        interaction_options.set_interaction_control_mode(args.interaction_control_mode)
+        interaction_options.set_in_endpoint_frame(args.in_endpoint_frame)
+        interaction_options.set_force_command(args.force_command)
+        interaction_options.set_K_nullspace(args.K_nullspace)
+        if len(args.interaction_frame) < 7:
```

Comment:
This is the same block of code that was used in the other class. Is there any way to avoid this code duplication?